docs/reference/commandline/daemon.md

```diff
@@ -616,3 +617,14 @@ set like this:
     # or
     export DOCKER_TMPDIR=/mnt/disk2/tmp
     /usr/local/bin/docker daemon -D -g /var/lib/docker -H unix:// > /var/lib/docker-machine/docker.log 2>&1
+
+# Cgroup parent
+
+You can specify cgroup parent for all containers with `--cgroup-parent` argument.
+There are two different ways to set it - a difference in leading `/`. If there is
+leading `/`, then cgroup will be created under root cgroup (e.g. for `/docker`:
+`/sys/fs/cgroup/memory/docker`), so it does not depend on daemon cgroup.
+Otherwise cgroup will be created under daemon cgroup (e.g. for `docker` if
+daemon is in `daemoncgroup` cgroup: `/sys/fs/cgroup/memory/daemoncgroup/docker`).
+By default this option is `/docker`. It can be changed per-container with
+`--cgroup-parent`argument for `create` or `run`.
```

Comment:
I had this draft, not really satisfied with it, but feel free to use if you think it reads better :smile:

```Markdown
# Default cgroup parent

The `--cgroup-parent` option allows you to set the default cgroup parent
to use for containers. If this option is not set, it defaults to `/docker`.

If the cgroup has a leading forward slash (`/`), the cgroup is created
under the root cgroup, otherwise the cgroup is created under the daemon
cgroup.

Assuming the daemon is running in cgroup `daemoncgroup`,
`--cgroup-parent=/foobar` creates a cgroup in
`/sys/fs/cgroup/memory/foobar`, wheras using `--cgroup-parent=foobar`
creates the cgroup in `/sys/fs/cgroup/memory/daemoncgroup/foobar`

This setting can also be set per container, using the `--cgroup-parent`
option on `docker create` and `docker run`, and takes precedence over
the `--cgroup-parent` option on the daemon. 
```